openedx/adg/lms/applications/tests/test_views.py

```diff
@@ -0,0 +1,446 @@
+"""
+Tests for all the views in applications app.
+"""
+
```

Comment:
No need for a blank line, Add blank line after class docstrings only as we have decided. Same goes for method docstrings.